analysis/hosting-stack/managed-1.txt

```diff
@@ -0,0 +1,20 @@
+https://site1-siteground.cmsdevrel.com/wp-content/index.php
```

Comment:
See above, if at all possible, it would be great to remove all those files with the concrete lists of URLs.

Reply:
Replied in the above comment.